release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -3607,3 +3607,19 @@ link:https://access.redhat.com/solutions/6960549[{product-title} 4.6.58 containe
 ==== Upgrading
 
 To upgrade an existing {product-title} 4.6 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster using the CLI] for instructions.
+
+[id="ocp-4-6-59"]
+=== RHBA-2022:4948 - {product-title} 4.6.59 bug fix and security update
+
+Issued: 2022-06-16
+
+{product-title} release 4.6.59, which includes security updates, is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2022:4948[RHBA-2022:4948] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2022:4947[RHSA-2022:4947] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/6963078[{product-title} 4.6.59 container image list]
+
+[id="ocp-4-6-59-upgrading"]
+==== Upgrading
+
+To upgrade an existing {product-title} 4.6 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster using the CLI] for instructions.
```

Comment:
LGTM! The only thing I'd point out is a missing anchor id for the adoc file: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#internal-cross-references. This is probably outside the scope of this PR, but other releases on this page are also missing anchor ids for this path. We can always do a followup PR. :)

Reply:
Agreed. Otherwise, LGTM!